### ISSUE TITLE
No sound for the APEX-SC System

### ISSUE THREAD

**Issue by user4-33** [musikinformatik/apex-sc-dafx#2]
When evaluating
`Apex2GUI.new(Server.default, "/path/to/apex-sc-dafx/data".resolveRelative);`
from DAFx15.scd the server and gui boots, but there is no sound when an audio output device was appropriately selected.
Also toggling the start/stop button doesn't help. The sliders do work as well as the values and vocal tract are changing.

What could I be missing here?

Supercollider 3.13.0
macOS 14.4.1

**Comment by telephon** [comment]
Yes, sorry, we didn't get the whole thing to work properly either, and didn't have the time to llok into it. So currently unsupported. Let's leave the issue open, if any more information is found.
